docs/pictures/zesong/DeleteWS.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
```

Comment:
<img width="325" alt="Screenshot 2020-10-29 at 9 17 35 AM" src="https://user-images.githubusercontent.com/46109197/97513603-b3c2b380-19c7-11eb-86b5-0b282fce6fda.png">

Hi, just take not that the picture is not loaded for diagram.

Reply:
![image](https://user-images.githubusercontent.com/50498839/97515588-51b87d00-19cc-11eb-8efc-f5094935948e.png)
Same issue for the UI component.